docs/image.rst

```diff
@@ -0,0 +1,108 @@
+Image
+*****
+
+.. py:module:: microbit
+
+Classes
+=======
+
+.. py:class:: Image(width_or_string=None, height=None, buffer=None)
+
+    If ``width_or_string`` is a string, creates an image from that string.
+
+    If it's an integer, a height has to also be specified, and it creates
+    an image with ``width_or_string`` columns and ``height`` rows. Optionally
+    ``buffer`` can be an array of ``width_or_string``×``height`` integers to
+    initialize the image.
```

Comment:
Having 2 constructors is fine, since they use different types for the args as well as different number of args, so it's very unlikely to make a mistake that doesn't raise an exception.

I'm happy to remove the buffer argument.  I added it at the start thinking it would be useful but I never actually used it.  Initialising with a string is the main way to create an image, but also being able to make a blank image of a given w*h is useful (and I use it).

We must remember that the target audience is children and things must stay simple.